src/solc_0.8/asset/ERC1155ERC721.sol

```diff
@@ -551,7 +522,7 @@ contract ERC1155ERC721 is WithSuperOperators, IERC1155, IERC721, ERC2771Handler
     /// @return URI string
     function tokenURI(uint256 id) public view returns (string memory) {
         require(_ownerOf(id) != address(0) || wasEverMinted(id), "NFT_!EXIST_||_FT_!MINTED");
```

Comment:
Is not part of this pr, but I think that is better if view functions don't revert but return a value to indicate failure

Reply:
The standard say that it's has to throw : https://eips.ethereum.org/EIPS/eip-721